setup.sh

```diff
@@ -1,56 +1,64 @@
 #!/bin/bash
-time=`date '+%Y%m%d_%H%M'`
-VAG_COMMON_DIR="/vagrant/"
+time=`TZ=Asia/Jerusalem date '+%Y%m%d_%H%M'`
+VAG_COMMON_DIR="/vagrant"
+HOME="/home/vagrant"
 LOG_DIR="$VAG_COMMON_DIR/log"
 OUTFILE="$LOG_DIR/${time}_vagrant-out.log"
+FLASK_APP_PY="$VAG_COMMON_DIR/flask_init.py"
+FLASK_DIR="$HOME/FlaskApp"
+FLASK_EXEC="$HOME/runFlask.sh"
+KILLFLASK_SCRIPT="$VAG_COMMON_DIR/killFlask.sh"
+PYTHON3="`which python3`"
+FLASK_ONBOOT=false # Currently Flask On Startup feature disabled [ Not Supported ]
 
 runSetup() {
     step=1
 
+    if [ ! -e $FLASK_APP_PY ]; then
+        echo "ERROR: Missing $FLASK_APP_PY file..." >> $OUTFILE
```

Comment:
you are writing to OUTFILE here, which is in the LOG_DIR
A few lines down you create LOG_DIR if it doesn't exist.

Reply:
Fix will be applied in next commit.